### ISSUE TITLE
Implementation of ISO 19115-3 Geospatial Metadata Standard

### ISSUE THREAD

**Issue by LyndaW** [GSA/datagov-wptheme#764]
Given the FGDC endorsement of ISO 19115-1 and the pending endorsement of the associated 19115-3 implementation schema, there is a need to enable Data.gov to process, validate, and display metadata formatted to the standard.

**Comment by kalxas** [comment]
At this moment the FGDC metadata are transformed into ISO-19115-1 during metadata ingestion to CKAN and CSW. I assume this request is for native support of ISO-19115-1 records.

**Comment by LyndaW** [comment]
Yes, native support to ingest ISO 19115-1:2014 metadata records. CSDGM (FGDC) metadata are currently transformed to ISO 19115-2:2009 (19115 extension).

**Comment by kalxas** [comment]
ah, sorry, you mean the 2014 version of the specification.

**Comment by theresaanna** [comment]
Internal note: This feels like an epic. What level of effort is required for implementation of what is described?

**Comment by kalxas** [comment]
This effort would require to implement ISO 19115-3 support in various places:
1. The Java app that data.gov uses to transform FGDC to ISO currently
2. In CKAN spatial extension
3. In OWSlib
4. In pycsw for the CSW endpoint
